gammapy/makers/utils.py

```diff
@@ -228,11 +277,8 @@ def make_map_background_irf(
         Background predicted counts sky cube in reconstructed energy.
     """
     # TODO:
-    #  This implementation can be improved in two ways:
-    #  1. Create equal time intervals between TSTART and TSTOP and sum up the
-    #  background IRF for each interval. This is instead of multiplying by
-    #  the total ontime. This then handles the rotation of the FoV.
-    #  2. Use the pointing table (does not currently exist in CTA files) to
+    #  This implementation can be improved in one ways:
+    #  Use the pointing table (does not currently exist in CTA files) to
```

Comment:
Indeed, for CTAO there is none. But for the HESS FITS production there is.
@LauraOlivera: is the POINTING HDU filled for HAWC/SWGO.

In Gammapy, we are currently only supporting the IACT array pointing info, ie at fixed position. This is handled with the class `FixedPointingInfo` and an `Observation` is build with such pointing class. So, the information is available (at worst, it is None).
So, either you can try to use such information in this PR, or do it into a subsequent PR.

Reply:
I don't plan on handling pointing tables, as it would probably require a lot more work if the source can change.
That is why I edited the TODO to remove only the first item that is covered by this PR.

Reply:
>  POINTING HDU filled for HAWC/SWGO.

Why would it be? HAWC / SWGO are completely fixed.

Reply:
> > POINTING HDU filled for HAWC/SWGO.
> 
> Why would it be? HAWC / SWGO are completely fixed.

In term of future formats, one should remind ourself that this HDU is not mandatory for non-pointing instrument. And then later, in our code, one will have 2 cases to handle